src/nl/hannahsten/texifyidea/lang/commands/LatexGenericRegularCommand.kt

```diff
@@ -143,7 +143,7 @@ enum class LatexGenericRegularCommand(
     INCLUDEFROM("includefrom", RequiredFolderArgument("absolute path"), RequiredFileArgument("filename", false, false, "tex"), dependency = LatexPackage.IMPORT),
     INPUT("input", RequiredFileArgument("sourcefile", true, false, "tex")),
     INPUTFROM("inputfrom", RequiredFolderArgument("absolute path"), RequiredFileArgument("filename", false, false, "tex"), dependency = LatexPackage.IMPORT),
-    INCLUDEGRAPHICS("includegraphics", "key-val-list".asOptional(), RequiredPicturePathArgument("imagefile", true, false, "pdf", "png", "jpg", "eps", "tikz"), dependency = GRAPHICX),
+    INCLUDEGRAPHICS("includegraphics", "key-val-list".asOptional(), RequiredPicturePathArgument("imagefile", true, false, "png", "pdf", "jpg", "jpeg", "jb2", "eps", "mps", "PNG", "PDF", "JPG", "JPEG", "JB2", "EPS"), dependency = GRAPHICX),
```

Comment:
So the order matters, but the code is using Set<String> everywhere. We should change that to List<String> so we actually try the extensions in the correct order

Reply:
Also, why does this exist as well?

https://github.com/Hannah-Sten/TeXiFy-IDEA/blob/13f9ef7aeba515d89ac08387cbeb01dea4a1c90c/src/nl/hannahsten/texifyidea/util/magic/FileMagic.kt#L63